### DISCUSSION TITLE
My station in Toronto (Etobicoke), ON, Canada

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#194] [Show and tell]
Hi all!
This is my station AIS, ADS-B:
![photo_2023-11-02_23-52-25](https://github.com/jvde-github/AIS-catcher/assets/20313019/468c39ae-222b-47cd-8561-2fda7728c3b1)

**Hardware** 
Inside BOX:
1.  OrangePi Zero2
2.  for ADS-B RTL-SDR dongle + LNA(21dB) with SAW (1090)
3.  for AIS RTL-SDR dongle + SAW (TB0436A)
4. 10W USB Power adapter 
5. 5v 4010 fan

RTL-SDR (R860) dongle inside (two identical):
![photo_2023-11-03_00-44-49](https://github.com/jvde-github/AIS-catcher/assets/20313019/96e389f4-f095-45ed-bac1-4b1cb2f8b316)

ADS-B > LNA with SAW 1090 (_SAW datasheet_: https://www.taisaw.com/assets/PDF/TA1090EC%20_Rev.3.0_.pdf):
![photo_2023-11-03_00-32-36](https://github.com/jvde-github/AIS-catcher/assets/20313019/d5d75a8c-0538-40a4-9f54-ec895cbacb11)

AIS > SAW 162MHz (_SAW TB0436A datasheet_: https://www.taisaw.com/assets/PDF/TB0436A%20_Rev.1.0_.pdf):
![photo_2023-11-03_00-35-09](https://github.com/jvde-github/AIS-catcher/assets/20313019/355be2db-1a5a-4744-8cc8-1b72ec93a7f7)

![d6090cb2f2883cc3e5b4d02d0aee7624d1dbedf0](https://github.com/jvde-github/AIS-catcher/assets/20313019/0b7ee8b8-2ac7-413d-82c9-2bc9b24763e2)
![46f2649b19f4c9bb69dc9340753572a38bdf7971](https://github.com/jvde-github/AIS-catcher/assets/20313019/4dcf466f-07c5-4cc1-9022-68a6382cfa2d)

The AIS flower pot antenna:
![AIS](https://github.com/jvde-github/AIS-catcher/assets/20313019/243075d0-223f-4d02-88bb-d37a3ef4d5f3)
![3bb2621de0a9bd22618e54dc3d5c348fe77a4c00](https://github.com/jvde-github/AIS-catcher/assets/20313019/3fc97da3-5b1c-4a1d-9984-f5a43e871be2)
calculator: https://vk2zoi.com/articles/half-wave-flower-pot/

Simple cheap antenna from Ali for ADS-B
![ADS-B](https://github.com/jvde-github/AIS-catcher/assets/20313019/1edf484b-8238-470f-b915-83f959fe3419)

It is installed on the balcony of a condo (approximately 50m above ground). Therefore, it has a limited overview of the horizon.

**Software** 
OS Debian, AIS-catcher, dump1090-fa and the feeders for many ADS-B aggregators.

_my AIS feeders:_
https://www.shipxplorer.com/stations/SXAISD0000204
https://www.myshiptracking.com/stations/station?id=736
https://www.marinetraffic.com/en/ais/details/stations/23355/_:9e7a5cc315c8f17f39f6afb8278e1656

_my ADS-B feeders:_
https://www.flightradar24.com/account/feed-stats/?id=18216
https://www.radarbox.com/stations/EXTRPI014534
https://www.flightaware.com/adsb/stats/user/Taxom#stats-159675
 
 
 **I will be happy to answer your questions. 73!**

**Comment by jvde-github** [comment]
Nice! Thanks for sharing.

**Reply by OE1MWW** [reply]
Hi Taxom,
do you have any source to buy the SAW TB0436A ?

73s

**Reply by Taxom** [reply]
Hi,
I bought it in Digikey, but now, unfortunately, this is no longer there.
https://www.digikey.ca/en/products/detail/tst/TB0436A/16906921

73.

**Comment by mdkendall** [comment]
Nice. It is interesting that you saw such an improvement with the addition of the 162MHz SAW, since it has 2.7dB loss. I guess there must be an out-of-band strong local interferer that overloads the RTL-SDR front end? Or maybe the RTL-SDR does not have good spurious response rejection?

**Reply by Taxom** [reply]
No, this is not so, after I installed the filter, I compensated this increase the gain. The SDR has very poor adjacent-channel selectivity, this is a common problem of such type receivers. 

And I have some powerful weather transmitting near AIS-channels.
![019203f541439592dca97c7652c1a8d827c456da](https://github.com/jvde-github/AIS-catcher/assets/20313019/4bc15d6f-5371-41e1-9b99-61f0b44a96b3)
![8d7015d0b561f85f322a65885feb860594966e7a](https://github.com/jvde-github/AIS-catcher/assets/20313019/c6bbf6d2-1fcb-4ad1-b014-9ba46c2f9d1b)


There are more about this in these two topics:
https://discussions.flightaware.com/t/howto-run-ads-b-receiver-ais-receiver-on-same-rpi/85040/121
https://discussions.flightaware.com/t/does-anyone-monitor-ais-ship-and-vessel-tracking/53634/408

**Reply by mdkendall** [reply]
Thanks. Those links are interesting. I will do some testing with a broadcast FM reject filter (I have one on hand) and I will see if I can find a source for a 162MHz SAW, or build an LC filter to experiment with.
